clang/test/Driver/rtsan.c

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang     -target x86_64-unknown-linux -fsanitize=realtime %s -S -emit-llvm -o - | FileCheck %s
```

Comment:
Driver tests should only use `-###` to test CC1 options. We don't test different `-O` combinations. It's not useful to test `-flto`

test/CodeGen tests `%clang_cc1`

We generally don't add end-to-end tests.

Reply:
The basis of this test file were very similar ones for asan and tsan:
https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/tsan.c
https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/asan.c

So I thought it was required for rtsan as well! Do you recommend I delete this test, or move it elsewhere? I defer to your expertise.

 I will fix the tests in Codegen to use clang_cc1

Reply:
Thx for letting me know the existing asan.c/tsan. I'll try to clean up the two tests.

For this driver test which will use `-###`, `-O` is entirely uninteresting.

Sometimes `-O1` or above may be interesting for CodeGen tests but probably not for this.

Reply:
Got rid of this test completely, thanks for the help understanding all this. I'll mark this as resolved